### DISCUSSION TITLE
Sugestão de novas tags

### DISCUSSION THREAD

**Discussion by leitoraincomum** [levxyca/diciotech#108] [Q&A]
Com base nas tags já existentes (Conceito, Mobile, Front-end, Back-end, Design), acredito que seja pertinente algumas situações:
- a tag de design talvez pode ser dividida em UI (user interface) e UX (user experience);
- inserir a tag de multiplataforma, já que alguns conceitos se aplicam a mais de um segmento;
- inserir a tag de projetos que se aplica a algumas ferramentas utilizadas em áreas de tecnologia.

**Comment by levxyca** [comment]
Primeiramente, obrigada pelas sugestões! 🤩

Como a ideia das tags é ser algo genérico mesmo, não sei se tem tanta necessidade subdivir em UI e UX, mas se acharem necessário eu não vejo problemas! Talvez entre até em uma divisão parecida com `front-end` e `back-end` né?

As tag de multiplataforma e de projetos acho super legais!

Vou deixar essa discussão aberta para ver se alguém da comunidade tem algum pitaco sobre :) aí podemos converter a discussion para issue novamente depois!

**Reply by leitoraincomum** [reply]
Ahhh Lev, pensando assim se a intenção é ser generalista, manter como está é bom sim.
Pensei mais em quem procura por especificidade mesmo.

Obrigada por me ler sempre, rs.

Beijo

**Reply by levxyca** [reply]
Estou sempre aberta a ouvir 🥰 a ideia é ter as tags de forma generalista e o campo de busca para quem procura por algum específico.
